Source/Scene/Model.js

```diff
@@ -2951,6 +2953,16 @@ define([
 
             var rs = rendererRenderStates[material.technique];
 
+            // reverse for negative scales
+            var nodeScales = gltfNode.scale;
+            var scalesProd = nodeScales[0] * nodeScales[1] * nodeScales[2];
```

Comment:
Ideally I think we'd want to use the `computedMatrix` in some way, but I'm still not sure how.

Some of the problems here might be if the gltf node's scale is undefined (if it's using a matrix), or if the node's parent is really the one with the negative scale.

Reply:
We could check if a node has TRS values or a matrix and then either take the product of scales for the TRS or the determinant for the matrix.
However, you are right about the parenting issue...checking the node itself is not sufficient.